languages/ruby/reference/exercise-concepts/resistor-color-duo.md

```diff
@@ -0,0 +1,51 @@
+# Concepts of Resistor Color Duo
+
+## General
+
+ [Example solution](https://github.com/exercism/ruby/blob/master/exercises/resistor-color-duo/.meta/solutions/resistor_color_duo.rb)
+
+- Classes:
+  - Defining classes
+- Methods:
+  - Defining methods
+  - Arguments
+  - Return values
+- Strings:
+  - Basics like difference between `1` and `"1"`
+- Class methods:
+  - How to define a method so it can be called directly on a class
+
+Depending on the approach either:
+- Hashes:
+  - How to initialize/create a hash
+  - How to access it's values
+
+or
+- Arrays:
+   - How to create an array
+   - What are indexes and how can they be useful
+   - How to get the index of a value that's stored in an array
+
+### Approach: use string interpolation (positional approach)
+
+- String interpolation
+- How to get the Integer representation of a String
```

Comment:
Removed the `#{}` and `#to_i` from here because, as you said, no reason to suggest specific methods